src/screens/qrScreen/qrScreen.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { Fragment, useRef, useState } from 'react';
+import { useIntl } from 'react-intl';
+import { View, Text, ScrollView, Button, TouchableOpacity, Dimensions } from 'react-native';
+import { BasicHeader, MainButton } from '../../components';
+import QRCodeScanner from 'react-native-qrcode-scanner';
+
+// styles
+import styles from './qrScreenStyle';
+import { useAppSelector } from '../../hooks';
+import postUrlParser from '../../utils/postUrlParser';
+import { getPost, getUser } from '../../providers/hive/dhive';
+import ROUTES from '../../constants/routeNames';
+import get from 'lodash/get';
+import parseAuthUrl from '../../utils/parseAuthUrl';
+import { navigate } from '../../navigation/service';
+
+const screenWidth = Dimensions.get('screen').width;
+const QRScreen = () => {
+  const intl = useIntl();
+  const currentAccount = useAppSelector((state) => state.account.currentAccount);
+
+  const [isActive, setIsActive] = useState(true);
+  const [scanResult, setScanResult] = useState(null);
+  const scannerRef = useRef(null);
+  const onSuccess = (e) => {
+    setScanResult(e.data);
+    setIsActive(false);
+  };
+
+  const _handleOpen = () => {
+    console.log('scanResult : ', scanResult);
+    _handleDeepLink(scanResult)
+  };
+
+  const _handleDeepLink = async (url = '') => {
```

Comment:
@aliseyalvi  We should reuse current implementation... perhaps more it to utils and use same method from both components.